design/user_types.go

```diff
@@ -32,11 +32,23 @@ var UpdateWorkItemPayload = Type("UpdateWorkItemPayload", func() {
 	Required("type", "fields", "version")
 })
 
+// CreateWorkItemTypePayload explains how input payload should look like
 var CreateWorkItemTypePayload = Type("CreateWorkItemTypePayload", func() {
 	Attribute("name", String, "Readable name of the type like Task, Issue, Bug, Epic etc.")
 	Attribute("fields", HashOf(String, fieldDefinition), "Type fields those must be followed by respective Work Items.")
 	Attribute("extendedTypeName", String, "If newly created type extends any existing type")
 	Required("name", "fields")
+	Example(map[string]interface{}{
```

Comment:
Any reason why this is defined as a Payload example, as oppose to individual examples of the Attribute like done else where?